### ISSUE TITLE
Taille de papier

### ISSUE THREAD

**Issue by cypriani** [patacrep/patacrep#248]
La [documentation](/patacrep/patacrep-doc/blob/master/book/layout.rst) dit que l'on peut changer la taille et l'orientation du papier, mais n'explique pas la marche à suivre. J'ai essayé d'utiliser `classoptions`, mais il semblerait que la taille du papier soit forcée par l'utilisation du package `geometry` dans le template `patacrep.tex`, sans option pour la changer. Y aurait-il moyen d'ajouter une option de template pour la taille du papier, ou faut-il que je crée un nouveau template personnalisé ?

**Comment by cypriani** [comment]
Bon j'ai réussi en utilisant le template data.tex au lieu de patacrep.tex. Il faudrait préciser dans la doc comment ça fonctionne, ainsi qu'une liste des options (ou une référence à la doc du paquet geometry). Je vais essayer de m'en occuper à l'occasion. Je suggère de laisser ce bug ouvert d'ici là (peut-on l'assigner au projet patacrep-doc ?).

**Comment by oliverpool** [comment]
Effectivement, la doc est loin d'être complète/à-jour... mais ton aide est la bienvenue !

J'essayerai d'être réactif pour tes PR/commentaires à l'avenir
